lib/isMobilePhone.js

```diff
@@ -42,6 +42,7 @@ var phones = {
   'pt-BR': /^(\+?55|0)\-?[1-9]{2}\-?[2-9]{1}\d{3,4}\-?\d{4}$/,
   'pt-PT': /^(\+?351)?9[1236]\d{7}$/,
   'ru-RU': /^(\+?7|8)?9\d{9}$/,
+  'ua-UA': /^(\+?3|8)?9\d{9}$/,
```

Comment:
You'll need to update `src/lib/isMobilePhone.js`. This file is compiled output from babel.